Subject: [PATCH] Fix pmac_zilog as console
From: Benjamin Herrenschmidt <benh@kernel.crashing.org>

http://linux.bkbits.net:8080/linux-2.6/gnupatch@418eeaabEopKEhWRlJFrp5KeVLAtag
...
By default, BREAK is used for sysrq, though some
Apple zilog's seem to have non-working BREAK detection logic, thus the
driver has a #define option you can manually enable when debugging to do
SYSRQ with ctrl-O instead (like pSeries virtual consoles).
...

Signed-off-by: Olaf Hering <olh@suse.de>

 drivers/serial/pmac_zilog.c |    2 +-
 1 files changed, 1 insertion(+), 1 deletion(-)

Index: linux-2.6.15-rc2-olh/drivers/serial/pmac_zilog.c
===================================================================
--- linux-2.6.15-rc2-olh.orig/drivers/serial/pmac_zilog.c
+++ linux-2.6.15-rc2-olh/drivers/serial/pmac_zilog.c
@@ -40,7 +40,7 @@
 
 #undef DEBUG
 #undef DEBUG_HARD
-#undef USE_CTRL_O_SYSRQ
+#define USE_CTRL_O_SYSRQ
 
 #include <linux/config.h>
 #include <linux/module.h>
